committeeoversightapp/models.py

```diff
@@ -9,17 +19,88 @@ class HearingCategoryType(models.Model):
     def __str__(self):
         return u'({0}) {1}'.format(self.id, self.name)
 
+
 class HearingCategory(models.Model):
     event = models.ForeignKey(Event, null=True, on_delete=models.CASCADE)
-    category = models.ForeignKey(HearingCategoryType, null=True, on_delete=models.CASCADE)
+    category = models.ForeignKey(HearingCategoryType,
+                                 null=True,
+                                 on_delete=models.CASCADE)
+
 
 class WitnessDetails(models.Model):
     witness = models.ForeignKey(EventParticipant, on_delete=models.CASCADE)
-    document = models.ForeignKey(EventDocument, null=True, blank=True, on_delete=models.CASCADE)
-    organization = models.CharField(max_length=100, null=True, blank=True, primary_key=False)
+    document = models.ForeignKey(EventDocument,
+                                 null=True,
+                                 blank=True,
+                                 on_delete=models.CASCADE)
+    organization = models.CharField(max_length=100,
+                                    null=True,
+                                    blank=True,
+                                    primary_key=False)
     retired = models.BooleanField(default=False)
 
+
 class Committee(models.Model):
     lugar_id = models.IntegerField(null=True, blank=True, primary_key=False)
-    lugar_name = models.CharField(max_length=200, null=True, blank=True, primary_key=False)
-    organization = models.ForeignKey(Organization, null=True, blank=True, on_delete=models.CASCADE)
+    lugar_name = models.CharField(max_length=200,
+                                  null=True,
+                                  blank=True,
+                                  primary_key=False)
+    organization = models.ForeignKey(Organization,
+                                     null=True,
+                                     blank=True,
+                                     on_delete=models.CASCADE)
+
+
+class StaticPage(Page):
```

Comment:
This is modeled on the LISC basic page template: https://github.com/datamade/lisc-cnda/blob/master/backend/cms/models.py#L10

This will be finished in a separate PR but was useful to get the Category pages working smoothly in the CMS backend.

Wagtail [StreamFields](https://docs.wagtail.io/en/v2.6.1/topics/streamfield.html) are very neat and allow the CMS user to put in an arbitrary number of different elements you define (in this case: headers, text blocks, images, and buttons) in whatever order they'd like. The `Page` model this inherits from also automatically provides a `title` attribute, so I didn't have to name it here but it will show up in the editing interface.